applications/GeoMechanicsApplication/custom_utilities/input_utility.cpp

```diff
@@ -0,0 +1,44 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "input_utility.h"
+#include "includes/model_part_io.h"
+#include "utilities/read_materials_utility.h"
+
+namespace Kratos {
+
+Parameters InputUtility::ProjectParametersFrom(const std::string& rProjectFilePath) const {
+    std::ifstream t(rProjectFilePath);
+    std::stringstream buffer;
+    buffer << t.rdbuf();
+    Parameters projFile{buffer.str()};
+    return projFile;
+}
+
+void InputUtility::ReadModelFromFile(const std::filesystem::path& rModelPartFilePath,
+                                            Kratos::ModelPart& rModelPart) const
```

Comment:
I prefer to align the type names and parameter names in a column-wise fashion, i.e.:
```suggestion
void InputUtility::ReadModelFromFile(const std::filesystem::path& rModelPartFilePath,
                                     Kratos::ModelPart&           rModelPart) const
```

(Perhaps is this something that we should add to our own coding conventions which supplement the Kratos Style Guide.)

Reply:
I agree it looks better, but when renaming anything, the alignment will be wrong again, so that's why I generally don't spend extra time for the 2nd column alignment, but keep it like it is after this commit. Let me know what you think 👍

Reply:
I suggest we should first come up with coding conventions in addition to the Kratos Style Guide before making this kind of stylistic changes.